src/test/javascript/spec/component/grading-system/grading-system.component.spec.ts

```diff
@@ -74,7 +74,7 @@ describe('Grading System Component', () => {
 
     beforeEach(() => {
         TestBed.configureTestingModule({
-            imports: [ArtemisTestModule, TranslateTestingModule, MockModule(FormsModule), RouterTestingModule.withRoutes([])],
+            imports: [ArtemisTestModule, MockModule(FormsModule)],
```

Comment:
I think you can remove the import of the `FormsModule` when you add
```
MockDirective(NgModel),
MockDirective(NgSelectOption),
```
to the declarations.

Reply:
Agree, resolved in 10f9bcd.